rules/windows/privilege_escalation_uac_bypass_dll_sideloading.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2020/10/27"
+maturity = "production"
+updated_date = "2020/10/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to bypass User Account Control (UAC) via DLL side-loading. Attackers may attempt to bypass UAC to stealthily execute code with
+elevated permissions.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "UAC Bypass Attempt via Privileged IFileOperation COM Interface"
+references = ["https://github.com/hfiref0x/UACME"]
+risk_score = 73
+rule_id = "5a14d01d-7ac8-4545-914c-b687c2cf66b3"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+file where event.type : "change" and process.name : "dllhost.exe" and
+
```

Comment:
prob dont need all the new lines in this query, but otherwise lgtm

Reply:
removed